frameworks/Ruby/grape/Gemfile

```diff
@@ -1,20 +1,21 @@
 source 'http://rubygems.org'
 
 platforms :jruby do
-  gem "activerecord-jdbcmysql-adapter", "~> 1.3.9", :require => false
-  gem 'torqbox', '0.1.7'
-  gem "trinidad", "1.4.6"
+  gem 'activerecord-jdbcmysql-adapter', '~> 1.3.9', :require => false
+  gem 'torquebox', '3.2.0'
```

Comment:
@rjnienaber we don't have `run_jruby_*` files for now, but in Sinatra I see https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Ruby/sinatra/Gemfile#L9
this is will be next step, add JRuby tests for all Ruby frameworks